bin/spotxBidAdapter.js

```diff
@@ -324,15 +313,15 @@ function outstreamRender(bid) {
       script.setAttribute('data-spotx_blocked_autoplay_override_mode', '1');
       script.setAttribute('data-spotx_video_slot_can_autoplay', '1');
 
-      if (bid.renderer.config.inIframe && window.document.getElementById(bid.renderer.config.inIframe).nodeName == 'IFRAME') {
-        const rawframe = window.document.getElementById(bid.renderer.config.inIframe);
+      if (inIframe && window.document.getElementById(inIframe).nodeName == 'IFRAME') {
+        const rawframe = window.document.getElementById(inIframe);
         let framedoc = rawframe.contentDocument;
         if (!framedoc && rawframe.contentWindow) {
           framedoc = rawframe.contentWindow.document;
         }
         framedoc.body.appendChild(script);
       } else {
-        window.document.getElementById(bid.video_slot).appendChild(script);
+        window.document.getElementById(videoSlot).appendChild(script);
```

Comment:
yep, this should be a slot, not a video slot... videoslot is usually an actual video element